index.src.html

```diff
@@ -2819,6 +2842,18 @@
     The {{VideoFrame/duration}} getter steps are to return
     {{VideoFrame/[[duration]]}}.
 
+: <dfn attribute for=VideoFrame>hasAlpha</dfn>
+:: Whether or not the {{VideoFrame}} has an alpha channel. If this is `false`,
```

Comment:
@sandersdan do you like "channel" or "component" for this? Same same?

Reply:
Channel is correct. A component is part of a sample but a plane has many samples.